packages/react-native/React/Base/macOS/RCTUIKit.m

```diff
@@ -245,6 +245,7 @@ @implementation RCTUIView
     self->_userInteractionEnabled = YES;
     self->_enableFocusRing = YES;
     self->_mouseDownCanMoveWindow = YES;
+    self->_transform3D = CATransform3DIdentity;
```

Comment:
Don't we need to read this somewhere in a `updateLayer` style method so we actually do the 3D transform?